src/AcceptanceTests/Infrastructure/TestIndependenceBehavior.cs

```diff
@@ -35,9 +36,9 @@ public TestIndependenceSkipBehavior(ScenarioContext scenarioContext)
 
         public Task Invoke(ITransportReceiveContext context, Func<ITransportReceiveContext, Task> next)
         {
-            if (!context.Message.Headers.TryGetValue("$AcceptanceTesting.TestRunId", out var runId) || runId != testRunId)
+            if (context.Message.Headers.TryGetValue("$AcceptanceTesting.TestRunId", out var runId) && runId != testRunId)
```

Comment:
What control messages are those that they were not needed in the past and need to be accounted for now?

Reply:
New control messages introduced with NSB 7.5 that acknowledge a successful retry to SC (see https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/ServicePlatform/Retries/RetryAcknowledgementBehavior.cs)